apis/constants.go

```diff
@@ -71,3 +71,25 @@ const (
 const (
 	CertificatePath = "/etc/vault/tls"
 )
+
+// Resource kind related constants
+const (
+	ResourceKindStatefulSet = "StatefulSet"
```

Comment:
Remove `ResourceKindStatefulSet` & `ResourceKindVaultServer` from here. These variables already exist in their respective API directory. Use those.

Reply:
Looks like only `ResourceKindVaultServer` was already present there, used that. Couldn't find any other occurrence of `ResourceKindStatefulSet`